1001-2000/LIT1295Dersan.xml

```diff
@@ -138,413 +146,1482 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                     <ab>
+                        <l n="127">፻፳፯፤ በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ፡ አመ፡ ዐሡሩ፡ ወሰኑዩ፡ ለወርኀ፡ ጥር፡ በዛቲ፡ ዕለት፡ ተዝካረ፡ በዓሉ፡ ለመልአክ፡ ክቡር፡
+                           ሚካኤል፡ ሊቀ፡ መላእክት፡ እስመ፡ በዛቲ፡ ዕለት፡ ፈነዎ፡ እግዚአብሔር፡ ለሚካኤል፡ ሊቀ፡ መላእክት፡ ኀበ፡ ያዕቆብ፡ እስራኤል፡ እንዘ፡ ይፈርህ፡ እኍሁ፡ ዔሳውሃ፡ ወኣድኃኖ፡
+                           እምኔሁ፡ ወአኅለፎ፡ ፈለገ፡ ዮርዳኖስ፡ ወሖረ፡ ኀበ፡ ላባ፡ እኍሃ፡ ለእሙ፡ ወአስተዋሰቦ፡ ክልኤ፡ አዋልዲሁ፡ ልያሃ፡ ወራሔልሃ፡ ወአግብኦ፡ እንዘ፡ ይጸይሕ፡ ፍኖቶ፡ በዳኅና፡ 
+                           ወበሰላም፡ ምስለ፡ ሰብኡ፡ ወንዋዩ፡ ወውሉዱ፡ ወተቀበሎ፡ እኍሁ፡ ዔሳው፡ በሰላም፡ ወፍቅር።</l>
+                        <l n="128">፻፳፰፤ ወበእንተ፡ ዝ፡ አዘዙነ፡ ከመ፡ ንግበር፡ ተዝካረ፡ በዓሉ፡ ለመልአክ፡ ክቡር፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ አመ፡ ዐሠሩ፡ ወሰኑዩ፡ ለለ፡ ወርኁ፡ ትንብል፡ ናሁ፡
+                           ወአስተብቍዖቱ፡ የሀሉ፡ ምስሌነ፡ ለዓለመ፡ አለም፡ አሜን።</l>
+                     </ab>
                </div>
-               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="SalamTerr">
-                  <label xml:lang="en">
-                     <foreign xml:lang="gez">Salām</foreign> for <foreign xml:lang="gez">Ṭǝrr</foreign>
-                  </label>
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="SalamEldest">
+                  <label xml:lang="en">Salām to you Michael, one of eldest highest, servant of the Lord</label>
+                  <!--This salām is transmitted in the section for Ṭǝrr in <ref type="mss" corresp="BAVet82"/>.-->
+                  <ab>
+                     <l n="129">፻፳፱፤ ሰላም፡ ለከ፡ ሚካኤል፡ አሐዱ፡ እምነ፡ ሊቃናት፡ ልዑላን፡ ለእግዚአብሔር፡ ፀማዱ፡ አመ፡ ተፈኖከ፡ ትኩን፡ ረድኤተ፡ ቴዎድሮስ፡ ጊዜ፡ መራዱ፡ ከመ፡ 
+                        ቈጽል፡ እምነፋስ፡ ሰብአ፡ ቍዝርዕዱ፡ ሶበ፡ መጥባሕተከ፡ ነጸሩ፡ በእዱ።</l>
+                  </ab>
                </div>
             </div>
             
             <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Yakkatit">
-               <label xml:lang="en">
-                  <foreign xml:lang="gez">Yakkātit</foreign>
-               </label>
-               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="HomilyYakkatit">
-                  <label xml:lang="en">Anonymous homily for <foreign xml:lang="gez">Yakkātit</foreign>
-                  </label>
-                  <listBibl type="editions">
+               <label xml:lang="en">Yakkātit</label>
+               <div type="textpart" subtype="part" xml:id="AdmonitionYakkatit">
+                  <label>Admonition for Yakkātit</label>
+                  <ab>በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ፡ ድርሳን፡ ዘቅዱስ፡ ወክቡር፡ ወኃያል፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ ዘይትነበብ፡ አመ፡ ዐሡሩ፡ ወሰኑዩ፡ ለወርኀ፡ 
+                     የካቲት። ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ ምስሌነ፡ ለዓለመ፡ ዓለም፡ አሜን።</ab>
+               </div>
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="HomilyPoor">
+                  <label xml:lang="en">Homily on a poor man of great faith</label>
+                  <!-->Anonymous in <ref type="mss" corresp="BMLacq776"/> and <ref type="mss" corresp="BAVet82"/>, but elsewhere as 
```

Comment:
If you mean that all between lines 568 and 576 must be commented out then better use `<!-- all text here will become green on you screen -->` but why would you want to hide all this?

Reply:
The selection of manuscripts for comparison is random and I think, it is not useful to present this information in an edition, that is based on a authorized publication.